tf2/CMakeLists.txt

```diff
@@ -92,9 +90,6 @@ if(BUILD_TESTING)
   ament_add_gtest(test_simple_tf2_core test/simple_tf2_core.cpp)
   if(TARGET test_simple_tf2_core)
     target_link_libraries(test_simple_tf2_core tf2)
-    ament_target_dependencies(test_simple_tf2_core
-      "geometry_msgs"
-    )
```

Comment:
This test does have a dependency on `geometry_msgs`, so shouldn't this change to a `target_link_libraries`?

Reply:
https://github.com/ros2/geometry2/pull/493#discussion_r840113294